integration-tests/src/test/java/com/redhat/parodos/flows/SimpleRestartWorkFlowTest.java

```diff
@@ -0,0 +1,509 @@
+package com.redhat.parodos.flows;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import com.redhat.parodos.flows.common.WorkFlowTestBuilder;
+import com.redhat.parodos.flows.common.WorkFlowTestBuilder.TestComponents;
+import com.redhat.parodos.sdk.api.WorkflowApi;
+import com.redhat.parodos.sdk.api.WorkflowDefinitionApi;
+import com.redhat.parodos.sdk.invoker.ApiException;
+import com.redhat.parodos.sdk.model.ArgumentRequestDTO;
+import com.redhat.parodos.sdk.model.WorkFlowContextResponseDTO;
+import com.redhat.parodos.sdk.model.WorkFlowDefinitionResponseDTO;
+import com.redhat.parodos.sdk.model.WorkFlowExecutionResponseDTO;
+import com.redhat.parodos.sdk.model.WorkFlowExecutionResponseDTO.WorkStatusEnum;
+import com.redhat.parodos.sdk.model.WorkFlowRequestDTO;
+import com.redhat.parodos.sdk.model.WorkFlowStatusResponseDTO;
+import com.redhat.parodos.sdk.model.WorkRequestDTO;
+import com.redhat.parodos.sdk.model.WorkStatusResponseDTO;
+import com.redhat.parodos.sdkutils.WorkFlowServiceUtils;
+import com.redhat.parodos.workflow.consts.WorkFlowConstants;
+import lombok.extern.slf4j.Slf4j;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Can we get rid of this annotation? IntelliJ sneaks it in, but it has no impact on the code itself.
If we wish to enforce, there are lombok annotations for nullity.